pages/linux/setquota.md

```diff
@@ -0,0 +1,24 @@
+# setquota
+
+> Set disk quotas for a user or group.
+> More information: <https://manned.org/setquota.8>.
+
+- Edit remote quota over RPC, uses rpc.quotad on the remote host to set quota:
+
+`setquota -r, --remote`
+
+- Set user quotas for named user:
+
+`setquota -u, --user`
```

Comment:
We can't create tldr pages only mention an isolated argument. The tldr pages are built with full working examples like the following:
```suggestion
`setquota --user {{username}} {{block_soft_limit}} {{block_hard_limit}} {{inode_soft_limit}} {{inode_hard_limit}} {{path/to/directory}}`
```

Reply:
Would resolve this now